index.js

```diff
@@ -1,84 +1,140 @@
-console.log("starting up!!");
-
+console.log("RUNNING INDEX.JS!!!");
 const express = require('express');
 const methodOverride = require('method-override');
 const pg = require('pg');
-
-// Initialise postgres client
 const configs = {
-  user: 'YOURUSERNAME',
+  user: 'kevin',
   host: '127.0.0.1',
   database: 'tunr_db',
   port: 5432,
 };
-
 const pool = new pg.Pool(configs);
-
-pool.on('error', function (err) {
-  console.log('idle client error', err.message, err.stack);
-});
-
-/**
- * ===================================
- * Configurations and set up
- * ===================================
- */
-
+pool.on('error',err=> console.log('idle client error', err.message, err.stack));
+/*===================================
+╔═╗┌─┐┌┬┐  ╦ ╦┌─┐
+╚═╗├┤  │   ║ ║├─┘
+╚═╝└─┘ ┴   ╚═╝┴
+===================================*/
 // Init express app
 const app = express();
-
-
 app.use(express.json());
-app.use(express.urlencoded({
-  extended: true
-}));
-
+app.use(express.urlencoded({extended: true}));
 app.use(methodOverride('_method'));
-
-
 // Set react-views to be the default view engine
 const reactEngine = require('express-react-views').createEngine();
 app.set('views', __dirname + '/views');
 app.set('view engine', 'jsx');
 app.engine('jsx', reactEngine);
-
-/**
- * ===================================
- * Routes
- * ===================================
- */
-
-app.get('/', (request, response) => {
-  // query database for all pokemon
-
-  // respond with HTML page displaying all pokemon
-  response.render('home');
+/*================================================
+╦  ┌─┐┌┐┌┌┬┐┬┌┐┌┌─┐
+║  ├─┤│││ │││││││ ┬
+╩═╝┴ ┴┘└┘─┴┘┴┘└┘└─┘
+================================================*/
+app.get('/',(req,res)=>{res.render('home')});
+/*================================================
+╦┌┐┌┌┬┐┌─┐─┐ ┬
+║│││ ││├┤ ┌┴┬┘
+╩┘└┘─┴┘└─┘┴ └─
+================================================*/
+app.get('/artists/',(req,res)=>{
+    text = 'SELECT * FROM artists ORDER BY id';
+    pool.query(text,(err,result)=>{
+        if (err) console.log(err);
+        res.render('index',result);
+    });
+});
+/*================================================
+╔╗╔┌─┐┬ ┬
+║║║├┤ │││
+╝╚╝└─┘└┴┘
+================================================*/
+app.get('/artists/new',(req,res)=>{res.render('new')});
+/*================================================
+╔═╗┬─┐┌─┐┌─┐┌┬┐┌─┐
+║  ├┬┘├┤ ├─┤ │ ├┤
+╚═╝┴└─└─┘┴ ┴ ┴ └─┘
+================================================*/
+app.post('/artists',(req,res)=>{
+    let {name, photo_url, nationality} = req.body;
+    text = `INSERT INTO artists (name, photo_url, nationality) VALUES ('${name}', '${photo_url}', '${nationality}') RETURNING *`
```

Comment:
dont use JS string interpolation, not good practice. use dollar sign like $1 instead